.github/workflows/ci.yml

```diff
@@ -279,14 +279,14 @@ jobs:
               run: vendor/bin/phpunit --order-by=reverse --extension Contao\\CoreBundle\\Tests\\PhpunitExtension\\GlobalStateWatcher
 
     nightly:
-        name: PHP 8.4
+        name: PHP 8.5
         runs-on: ubuntu-latest
         continue-on-error: true
         steps:
             - name: Setup PHP
               uses: shivammathur/setup-php@v2
               with:
-                  php-version: 8.4
+                  php-version: 8.5
```

Comment:
Can't we just use `nightly` so we don't have to update this every year? Then we'd benefit from what `setup-php` does: https://github.com/shivammathur/setup-php/blob/4a4c5a48bb842514513e89c6704a3a2e8b815f6c/src/configs/php-versions.json#L5

Reply:
Changed in 0e23a38bee502ac17b1719052d879b9a35876f33